### PR TITLE
viziphant-new (except UEA)

### PR DESCRIPTION
**Pull request by dizcza**

### CODE REVIEW

File: doc/acknowledgments.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-****************
+***************
 Acknowledgments
-****************
+***************
 
 This open source software code was developed in part or in whole in the Human Brain Project, funded from the European Union’s Horizon 2020 Framework Programme for Research and Innovation under Specific Grant Agreements No. 720270 and No. 785907 (Human Brain Project SGA1 and SGA2).
```
Comment (mdenker) [inline doc/acknowledgments.rst:5]:
```suggestion
This open source software code was developed in part or in whole in the Human Brain Project, funded from the European Union’s Horizon 2020 Framework Programme for Research and Innovation under Specific Grant Agreements No. 720270, No. 785907 and No. 945539 (Human Brain Project SGA1, SGA2 and SGA3).
```

File: LICENSE.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015, INM-6 & IAS-6
+Copyright (c) 2015-2020, INM-6 & IAS-6
```
Comment (mdenker) [inline LICENSE.txt:1]: For consistency with the commits, let's change the date to 2017:
```suggestion
Copyright (c) 2017-2020, Institute of Neuroscience and Medicine (INM-6), Forschungszentrum Jülich
```

File: doc/conf.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Viziphant'
+authors = 'Elephant team'
+copyright = u"2019-{this_year}, {authors}".format(this_year=date.today().year,
```
Comment (mdenker) [inline doc/conf.py:22]:
```suggestion
copyright = u"2017-{this_year}, {authors}".format(this_year=date.today().year,
```

File: setup.py [hunk 4]
```diff
@@ -12,27 +12,33 @@
...
     install_requires=install_requires,
+    extras_require=extras_require,
     author="Viziphant authors and contributors",
     author_email="contact@python-elephant.org",
     description="Viziphant is a package for the visualization of the analysis"
                 "results of electrophysiology data in Python",
     long_description=long_description,
     license="BSD",
-    url='https://github.com/INM-6/viziphant',
+    url='https://github.com/INM-6/viziphant-new',
```
Comment (mdenker) [inline setup.py:34]: Shouldn't this be 
```suggestion
    url='https://github.com/INM-6/viziphant',
```

File: viziphant/__init__.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+"""
+Viziphant is a package for the visualization of the analysis results from
+Elephant, which is a package for the analysis of neurophysiological data,
+based on Neo.
+"""
+# Copyright 2019-2020 by the Viziphant team, see `doc/authors.rst`.
```
Comment (mdenker) [inline viziphant/__init__.py:6]:
```suggestion
# Copyright 2017-2020 by the Viziphant team, see `doc/authors.rst`.
```

File: doc/conf.py (hunk 3 above)
Comment (dizcza) [reply doc/conf.py:22]: shouldn't it be 2015, as in the license?

**Pull request merged**